### PR TITLE
Added Volume Weighted Moving Average

### PR DESCRIPTION
**Pull request by brhurley**
Fixes #.

Changes proposed in this pull request:
- Adding VWMAIndicator to the indicators.averages package 

- [X] added an entry _with related ticket number(s)_ to the unreleased section of `CHANGELOG.md`

### CODE REVIEW

**Comment by brhurley** [comment]
Ready for review

**Comment by brhurley** [comment]
@nimo23 When you get time can you review this?

File: ta4j-core/src/test/java/org/ta4j/core/indicators/averages/VWMAIndicatorTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+public class VWMAIndicatorTest extends AbstractIndicatorTest<Indicator<Num>, Num> {
+
+    public VWMAIndicatorTest(NumFactory numFactory) {
+        super(numFactory);
+    }
+
+    @Test
+    public void vwmaIndicatorTest() {
+
+        MockIndicator mock = CsvTestUtils.getCsvFile(VWMAIndicatorTest.class, "VWMA.csv", numFactory);
```
Comment (TheCookieLab) [inline ta4j-core/src/test/java/org/ta4j/core/indicators/averages/VWMAIndicatorTest.java:124]: Please flesh out these unit tests further. You can use the CsvTestUtils however they should be in addition to (not a replacement for) atomic test cases including edge conditions. 

When testing using mock data, there should be at minimum one unit test that demonstrates the calculation using simple numbers so that other users can easily reason about the indicator's logic at a glance (as opposed to digesting an entire CSVs worth context).

Please see `RecentSwingLowIndicatorTest `or `KalmanFilterIndicatorTest` as examples of how I expect indicator unit tests to be structured.

**Review by TheCookieLab — CHANGES_REQUESTED** [review]
The PR's opening "post" should be filled out correctly. The template exists for a reason and all contributors are expected to do or provide reason why it's not applicable.

**Comment by brhurley** [comment]
> The PR's opening "post" should be filled out correctly. The template exists for a reason and all contributors are expected to do or provide reason why it's not applicable.

I added a line in the PR but please update your template on what you expect.   I couldn't find a CHANGE.md so I updated the CHANGELOG.md file.   Also while I don't see any other examples of ticket numbers in the CHANGELOG.md file I put the PR reference number on my line.

I'll work on adding a unit test like you requested next.

**Comment by TheCookieLab** [comment]
Gotcha, thanks,

**Comment by brhurley** [comment]
@TheCookieLab updated as requested.

**Review by TheCookieLab — APPROVED** [review]
